src/Sulu/Component/Content/Compat/DataNormalizer.php

```diff
@@ -66,6 +66,8 @@ public static function normalize(FormEvent $event)
             'shadowLocaleEnabled' => self::getAndUnsetValue($data, 'shadowOn') ? true : false,
             'parent' => self::getAndUnsetValue($data, 'parent'),
             'workflowStage' => self::getAndUnsetValue($data, 'workflowStage'),
+            'lastModifiedEnabled' => self::getAndUnsetValue($data, 'lastModifiedEnabled'),
+            'lastModified' => self::getAndUnsetValue($data, 'lastModified'),
```

Comment:
Is `lastModifiedEnabled` required here? What das toe DataNormalizer do is it responsible for writing the Admin API response? where data `$data` come from?